package.json

```diff
@@ -9,7 +9,9 @@
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack -p",
     "clean": "rimraf dist/",
-    "dev": "cross-env NODE_ENV=development webpack-dev-server -d --inline --hot --progress --no-info --port 3000",
+    "dev:server": "cross-env NODE_ENV=development nodemon --use_strict server/node-server.js --watch api server",
```

Comment:
Only watch /api and /server. 

Prevents rebuilding on src changes (webpack will handle this for us)